schemas/deviceToCloud/alert/alert.json

```diff
@@ -0,0 +1,37 @@
+{
+    "title": "Alert",
+    "description": "The device Alert",
+    "type": "object",
+    "properties": {
+      "appId": {
+          "type": "string",
+          "const": "ALERT"
+      },
+      "ts": {
+          "type": "number",
+          "description": "Unix timestamp given in milliseconds when the data was sampled"
+      },
+      "seq": {
+          "type": "number",
+          "description": "Sequence number of the message. This is a counter, used for ordering message if `ts` not available/used"
+      },
+      "desc": {
+          "type": "string",
+          "description": "Description of the alert"
+      },
+      "type": {
+          "type": "number",
+          "format": "int32",
+          "description": "Enmeration for the kind of alert. This value can be user defined. TBD: the range for custom values"
```

Comment:
When we have some common defined values (battery level, fall detected, etc) where would they be found?

Reply:
Should we just say that the reserved cloud defined values are 0-99 and user range is 100 and up?

Reply:
That seems good to me. @plskeggs What do you think?

Reply:
> When we have some common defined values (battery level, fall detected, etc) where would they be found?

This seems like something we need. I am not sure if this should be documented here or not. For now we have no "special" alerts so there is nothing to document.

Reply:
I am working on the device side code for this.  This aligns with what I've done.

```
/** @brief Alert type */
enum nrf_cloud_alert_type {
	/** Placeholder; not meant to be sent. */
	ALERT_TYPE_NONE,
	/** Device finished booting. */
	ALERT_TYPE_DEVICE_NOW_ONLINE,
	/** Device is shutting down. */
	ALERT_TYPE_DEVICE_GOING_OFFLINE,
	/** Operating temperature outside of limits. */
	ALERT_TYPE_TEMPERATURE,
	/** Battery level too low. */
	ALERT_TYPE_BATTERY,
	/** User specific alerts start here. */
	ALERT_TYPE_CUSTOM = 100
};

Reply:
Note -- in case it isn't widely known, enums in C automatically start at 0, and increase by 1 as one goes down the list, unless an optional `= N` is present, which will initialize that enum to N, and subsequent ones increment by 1 from there.

Reply:
I added some more to the list above.

Reply:
@plskeggs I will update the docs with your current set of enumerated values based on the above comment/code